scss/footer.scss

```diff
@@ -1,5 +1,6 @@
-@import 'colors';
-@mixin white-text {
+@import 'variables/colors';
+@import 'variables/sizes';
```

Comment:
If you are importing the variables files in the main `application.scss`, there's no need for you to import them here too! :smile_cat: